src/protocol/encoder.js

```diff
@@ -8,7 +8,7 @@ const INT64_SIZE = 8
 const MOST_SIGNIFICANT_BIT = 0x80 // 128
 const OTHER_BITS = 0x7f // 127
 const UNSIGNED_INT32_MAX_NUMBER = 0xffffff80
-const UNSIGNED_INT64_MAX_NUMBER = Long.fromBytes([-1, -1, -1, -1, -1, -1, -1, -128])
+const UNSIGNED_INT64_MAX_NUMBER = Long.fromNumber(-128)
```

Comment:
I'm not sure why this variable is called `UNSIGNED_INT64_MAX_NUMBER`, because that's definitely not `-128`. `-128` is the lowest number that possible to represent with an 8 bit signed integer, but I have long forgotten how exactly this piece of code works...

Reply:
Changed it to `0xffffffffffffff80n`, which makes a hell of a lot more sense than -128. No clue why it worked with -128 to begin with...

Reply:
I will 10h to explain this, jokes aside, it's hard to remember the details of number based things. I called this `*_MAX_NUMBER`, but I don't remember why anymore...

Reply:
Yeah, it's something about calculating the size of a varlong. I tried to find the documentation that we read when doing the initial implementation, to see if I could find a clue, but I couldn't find them.